torch/onnx/_internal/jit_utils.py

```diff
@@ -98,6 +99,49 @@ def aten_op(self, operator: str, *args, overload_name: str = "", **kwargs):
             **kwargs,
         )
 
+    @_beartype.beartype
+    def onnxscript_op(
+        self,
+        onnx_fn,  # TODO(titaiwang): annotate this when onnx-script becomes dependency
+        *raw_args: Union[torch.Tensor, _C.Value],
+        outputs: int = 1,
```

Comment:
I wonder if we could None here and infer that value from the annotation of the function and only specify it when it is unknown (like Split operator).

Reply:
I tried with ast and inspect package, just like onnx-script does, but it turns out at this point the function is no longer python-function, and it's ONNXfunction instead, which is not understandable by inspect. Digging more into this seems to be complicated for this goal... Do you have any sugestion on this?

Reply:
Attribute function should give ou access the the python function: https://github.com/microsoft/onnx-script/blob/main/onnxscript/values.py#L158.

Reply:
On the other hand, the outputs is passed through directly to `_add_op` and this seems to be following the same pattern as the other `op` methods above. So, maybe it doesn't matter? I wouldn't suggest analyzing the python-source-ast, it is best to leave that to ONNXScript (and just expose methods on ONNXFunction to return it's type-annotation (if available)).

Reply:
Yes, it makes sense to keep this as the same format as `op` API. I think I will leave this one as it is.